src/main/resources/application.properties

```diff
@@ -0,0 +1,95 @@
+###############
+# Persistence #
+###############
+
+
+# Tells the Kogito runtime what kind of persistence is configured.
+kogito.persistence.type=jdbc
+
+# BAMOE provides a separate Flyway subsystem that allows Kogito's internal tables to be created and migrated
+# without interfering with your application's Flyway, if you're using it.
+#
+# This property enables/disables running this isolated Flyway mechanism automatically when the application starts.
+# If you want more control over when/how/who executes changes to your database, disable it and run the SQL scripts manually.
+%prod.kie.flyway.enabled=false
+
+%prod.quarkus.datasource.db-kind=postgresql
+%prod.quarkus.datasource.username=kogito-user
+%prod.quarkus.datasource.password=kogito-pass
+%prod.quarkus.datasource.jdbc.url=${QUARKUS_DATASOURCE_JDBC_URL:jdbc:postgresql://0.0.0.0:5432/kogito}
+%prod.quarkus.datasource.reactive.url=${QUARKUS_DATASOURCE_REACTIVE_URL:postgresql://0.0.0.0:5432/kogito}
+
```

Comment:
Add a comment pointing to BAMOE Documentation for users to check other database options they'd have?

Reply:
I don't think we have that yet. But we can add it once it's available.

Reply:
Added a link to the BAMOE documentation home page